server/interceptor/cors/cors.go

```diff
@@ -14,6 +14,28 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
```

Comment:
Please remove the Apache License header as we can only put the Apache License header to the code which we have the copyright.